ethcore/vm/src/env_info.rs

```diff
@@ -20,9 +20,10 @@ use std::cmp;
 use std::sync::Arc;
 use hash::keccak;
 use ethereum_types::{U256, H256, Address};
-use types::BlockNumber;
 use ethjson;
 
+type BlockNumber = u64;
```

Comment:
Is including deps on common-type for vm an issue ?

Reply:
I think it's pretty redundant since we only need to import a typedef for `u64`. It doesn't really bring any value imho.
Maybe we can put `BlockNumber` into `ethereum-types`?